deployer/README.md

```diff
@@ -101,91 +167,51 @@ The following files may be supplied in the deployer secret:
-browsers will be directed for authenticating to OpenShift. You should
-read the [ElasticSearch](#elasticsearch) section below before choosing
-ElasticSearch parameters for the deployer. These and other parameters
-are available:
-
-* `KIBANA_HOSTNAME` (required): External hostname where web clients will reach Kibana
-* `PUBLIC_MASTER_URL` (required): External URL for the master, for OAuth purposes
-* `ES_CLUSTER_SIZE` (required): How many instances of ElasticSearch to deploy. At least 3 are needed for redundancy, and more can be used for scaling.
-* `ES_INSTANCE_RAM`: Amount of RAM to reserve per ElasticSearch instance (e.g. 1024M, 2G). Defaults to 8GB; must be at least 512M (Ref.: [ElasticSearch documentation](https://www.elastic.co/guide/en/elasticsearch/guide/current/hardware.html#_memory).
-* `ES_PVC_SIZE`: Size of the PersistentVolumeClaim to create per ElasticSearch ops instance, e.g. 100G. If empty, no PVCs will be created and emptyDir volumes are used instead.
-* `ES_PVC_PREFIX`: Prefix for the names of PersistentVolumeClaims to be created; a number will be appended per instance. If they don't already exist, they will be created with size `ES_PVC_SIZE`.
-* `FLUENTD_NODESELECTOR`: The nodeSelector to use for the Fluentd DaemonSet. Defaults to "logging-infra-fluentd=true".
-* `ES_NODESELECTOR`: Specify the nodeSelector that Elasticsearch should be use (label=value)
-* `KIBANA_NODESELECTOR`: Specify the nodeSelector that Kibana should be use (label=value)
-* `CURATOR_NODESELECTOR`: Specify the nodeSelector that Curator should be use (label=value)
-* `ENABLE_OPS_CLUSTER`: If "true", configure a second ES cluster and Kibana for ops logs. (See [below](#ops-cluster) for details.)
-* `KIBANA_OPS_HOSTNAME`, `ES_OPS_INSTANCE_RAM`, `ES_OPS_PVC_SIZE`, `ES_OPS_PVC_PREFIX`, `ES_OPS_CLUSTER_SIZE`, `ES_OPS_NODESELECTOR`, `KIBANA_OPS_NODESELECTOR`, `CURATOR_OPS_NODESELECTOR`: Parallel parameters for the ops log cluster.
-* `IMAGE_PREFIX`: Specify prefix for logging component images; e.g. for "docker.io/openshift/origin-logging-deployer:v1.1", set prefix "docker.io/openshift/origin-"
-* `IMAGE_VERSION`: Specify version for logging component images; e.g. for "docker.io/openshift/origin-logging-deployer:v1.1", set version "v1.1"
-* `IMAGE_PULL_SECRET`: Specify the name of an existing pull secret to be used for pulling component images from an authenticated registry.
-* `INSECURE_REGISTRY`: Allow the registry for logging component images to be non-secure (not secured with a certificate signed by a known CA)
-
 You run the deployer by instantiating a template. Here is an example with some parameters:
 
     $ oc new-app logging-deployer-template \
-               -p KIBANA_HOSTNAME=kibana.example.com \
-               -p ES_CLUSTER_SIZE=1 \
-               -p ES_INSTANCE_RAM=1G \
-               -p PUBLIC_MASTER_URL=https://localhost:8443
+               -p IMAGE_VERSION=v1.2.0 \
+               -p MODE=install
```

Comment:
I think that's worth mentioning.

On Fri, Jun 17, 2016 at 11:54 AM, Eric Wolinetz <notifications@github.com>
wrote:

> In deployer/README.md
> <https://github.com/openshift/origin-aggregated-logging/pull/158#discussion_r67533277>
> :
>
> >  You run the deployer by instantiating a template. Here is an example with some parameters:
> >
> >      $ oc new-app logging-deployer-template \
> > -               -p KIBANA_HOSTNAME=kibana.example.com \
> > -               -p ES_CLUSTER_SIZE=1 \
> > -               -p ES_INSTANCE_RAM=1G \
> > -               -p PUBLIC_MASTER_URL=https://localhost:8443
> > +               -p IMAGE_VERSION=v1.2.0 \
> > +               -p MODE=install
>
> Should we mention that this is the default mode or do we want to encourage
> users to be explicit?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/158/files/e059ad9d51cf7054b361ee80481b50a9ee001971#r67533277>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABz-js3_pdYlQEgxmkn_7QHTigy79GCks5qMsM-gaJpZM4Ir7Nu>
> .
>